care/facility/models/notification.py

```diff
@@ -23,6 +23,7 @@ class Medium(enum.Enum):
 
     class Event(enum.Enum):
         MESSAGE = 0
+        PUSH_MESSAGE = 220
```

Comment:
we can use a value between 0 and 20, or move this below PATIENT_NOTE_ADDED

Reply:
I felt the same way, to use a value between 0 and 20, but hesitated thinking that if we did have any previous event types which occupied that range and were later removed, new notifications under the same range would have a conflicting event type with those old notification data. I did not spend much time to go through and verify that, instead i found using 220 to be simpler, I've moved this event type to the bottom.